### PR TITLE
Fixed more linting errors

### PR DESCRIPTION
**Pull request by b12f**

### CODE REVIEW

File: frontend/src/app/shared/directives/a11y/double-click-or-tap.directive.ts [hunk 1]
```diff
@@ -31,12 +31,12 @@ import {
 } from '@angular/core';
 
 @Directive({
-  selector: '[doubleClickOrTap]',
```
Comment (HDinger) [inline frontend/src/app/shared/directives/a11y/double-click-or-tap.directive.ts:34]: I thought we want to remove it, as it is not used?

File: frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export class CKEditorSetupService {
     return this
       .createWatchdog(editorClass, contentWrapper, config)
       .then((watchdog:ICKEditorWatchdog) => {
-        const editor = watchdog.editor;
```
Comment (HDinger) [inline frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts:63]: I have no possibility to test this right now. @oliverguenther does the watchdog feature still work with this changes?

File: frontend/src/app/shared/directives/autofocus/autofocus.directive.ts [hunk 3]
```diff
@@ -4,10 +4,10 @@ import {
 import { FocusHelperService } from 'core-app/shared/directives/focus/focus-helper';
 
 @Directive({
-  selector: '[autoFocus]',
+  selector: '[opAutoFocus]',
```
Comment (HDinger) [inline frontend/src/app/shared/directives/autofocus/autofocus.directive.ts]: What about merging this with the `focus` directive? However, I am also fine with doing it separately.

**Review by HDinger — COMMENTED** [review]
Good job, I have only small remarks/questions. 👍 

However, the build crashes for me in the `wp-graph-configuration.service.ts`. Does this happen for you, too?

**Review by oliverguenther — APPROVED** [review]

**Comment by oliverguenther** [comment]
The changes in linter config LGTM. It seems you broke some features however

**Pull request merged**